src/index.ts

```diff
@@ -588,16 +603,39 @@ export class Bigtable {
     );
   }
 
+  getInstances(callback: GetInstancesCallback): void;
+  getInstances(gaxOptions: CallOptions, callback: GetInstancesCallback): void;
+  getInstances(gaxOptions?: CallOptions): Promise<GetInstancesResponse>;
```

Comment:
This has always been kinda fuzzy for me - is there any harm in declaring this AFTER the callback overloads? I know the official docs say to order them from most specific to least, but I don't really know if it matters here.